django/db/migrations/state.py

```diff
@@ -112,11 +112,11 @@ def remove_model(self, app_label, model_name):
 
     def rename_model(self, app_label, old_name, new_name):
         # Add a new model.
-        renamed_model = self.models[app_label, old_name.lower()].clone()
+        renamed_model = self.models[app_label, old_name].clone()
         renamed_model.name = new_name
```

Comment:
If we will take new_name_lower then this will also be changed to the lower form but we don't need that here so in `rename_model` shall we just pass old_name and new_name in spite of old_name_lower and new_name_lower. This is the reason of failing tests.

Reply:
That makes sense, I think we'll need to keep passing the case aware names in this case for `rename_model`.

Reply:
Committed the changes 😃